docs/user/build-customization.rst

```diff
@@ -196,3 +196,22 @@ It's possible to use ``post_checkout`` user-defined job for this.
          - ./git-lfs fetch
          # Make local files to have the real content on them
          - ./git-lfs checkout
+
+
+Document your code with ``jsdoc``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
This might be better off generalized, rather than specific to jsdoc. There is a lot more to say about jsdoc usage, and duplicating that here will probably guide users to the wrong guide content.

```suggestion
Install Node.js dependencies
~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
I like this idea, :100: